src/main/java/ai/elimu/web/content/multimedia/image/ImageCreateController.java

```diff
@@ -133,6 +135,11 @@ public String handleSubmit(
                         }
                     }
                 }
+
+                if (bytes.length > MAX_MB) {
+                    result.rejectValue("bytes", "image.too.big");
```

Comment:
Clarify that we are referring to the file size, not image (width/height):
```suggestion
                    result.rejectValue("bytes", "file.size.too.big");
```